packages/interaction/global.d.ts

```diff
@@ -0,0 +1,60 @@
+declare type InteractionTrackingData = any;
+
+declare type Cursor = 'auto'
```

Comment:
sorcery

Reply:
Those particular lines were moved it from globals. They will be removed as soon as someone converts interaction to TS :)